client/src/components/Filters/PriceFilter.js

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import ToggleButton from "@mui/material/ToggleButton";
+import ToggleButtonGroup from "@mui/material/ToggleButtonGroup";
+
+const DEFAULT_FILTER = {
+  $: false,
+  $$: false,
+  $$$: false,
+  $$$$: false,
+};
+
+function PriceCategoryFilter(restaurant) {
+  return selectedPrices.includes(restaurant.price_category);
+}
+
+export default function PriceFilterToggler() {
+  const [priceFilter, setPriceFilter] = React.useState(DEFAULT_FILTER);
+  const [filters, setFilters] = React.useState(() => []);
+
+  const handleFormat = (event, newFormats) => {
+    setFilters(newFormats);
+  };
```

Comment:
what is format / newFormats?

Reply:
This is for the toggle button group component, it doesnt work without this portion